package.json

```diff
@@ -44,15 +44,15 @@
     "commit": "ui-scripts --commit",
     "install:react": "ui-scripts --install-react",
     "install:packages": "lerna bootstrap --reject-cycles",
-    "bootstrap": "npm-run-all clean:modules install:packages clean build build:tokens build:types",
+    "bootstrap": "./scripts/bootstrap.sh",
```

Comment:
These 2 are not doing the same thing, I think the new script should delete the same folders too. Also instead of a shell script a .js "cleanAll" script would be better (unless the shell script has much better performance.)

Reply:
I rewrote the scripts to nodejs, and although they seem to be a lit slower it is still much faster than the previous solution.